pim-tool-back/src/main/java/pilotproject/Project_Management_ELCA/controller/GroupController.java

```diff
@@ -0,0 +1,37 @@
+package pilotproject.Project_Management_ELCA.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pilotproject.Project_Management_ELCA.model.dto.GroupDto;
+import pilotproject.Project_Management_ELCA.model.dto.ProjectDto;
+import pilotproject.Project_Management_ELCA.service.impl.GroupServiceImpl;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/group")
+@CrossOrigin(origins = "http://localhost:4200")
+public class GroupController extends AbstractApplicationController{
+
+    private final GroupServiceImpl groupServiceImpl;
```

Comment:
[S] Just use `GroupService` here, there is no point to create an interface if you inject the implementation directly.